draft-kleidl-digest-fields-problem-types.md

```diff
@@ -78,25 +78,71 @@ interpreted as described in {{PROBLEM}}.
 
 ## Unsupported Hashing Algorithm
 
-This section defines the "https://iana.org/assignments/http-problem-types#unsupported-hashing-algorithm" problem type. A server MAY use this problem type when responding to a request, whose integrity or integrity preference fields reference a hashing algorithm that the server can not or does not want to support for this request, and if the server wants to indicate this problem to the sender.
+This section defines the "https://iana.org/assignments/http-problem-types#unsupported-hashing-algorithm" problem type.
+A server MAY use this problem type if it wants to communicate to the client that
+none of the hashing algorithms referenced in the integrity or integrity preference fields present in the request,
+is supported.
 
-For this problem type, the `unsupported-algorithm` is defined as the only extension member. It SHOULD be populated in a response using this problem type, with its value being the algorithm key of the unsupported algorithm from the request. The response SHOULD include the corresponding integrity preference field to indicate the server's algorithm support and preference.
+For this problem type, the `unsupported-algorithm` is defined as the only extension member.
+It SHOULD be populated in a response using this problem type, with its value being the algorithm key of the unsupported algorithm from the request.
+The response SHOULD include the corresponding integrity preference field to indicate the server's algorithm support and preference.<!-- I am currently not sure whether to use normative language here. -->
```

Comment:
Do you see a reason why this shouldn't use normative language here?

Reply:
We can clarify that IF you send a problem-type response, you SHOULD then send an integrity preference field.
In this case, we are not only defining a problem-type, but even defining an associate behavior wrt to the fields that need to be used in conjunciton with it.

Reply:
Alright, then let's drop the normative language for now.